android/uhabits-core/src/main/java/org/isoron/uhabits/core/utils/DateUtils.java

```diff
@@ -175,9 +175,14 @@ public static long getStartOfToday()
         return getStartOfDay(getLocalTime());
     }
 
+    public static long getTomorrowStart()
```

Comment:
For consistency, please rename this to `getStartOfTomorrow()`

Reply:
done!